letsencrypt/plugins/manual_test.py

```diff
@@ -61,7 +61,28 @@ def test_perform(self, mock_raw_input, mock_verify, mock_urandom,
             self.achalls[0].challb.chall, "foo.com", KEY.public_key(), 4430)
 
         message = mock_stdout.write.mock_calls[0][1][0]
-        self.assertTrue(self.achalls[0].token in message)
+        self.assertEqual(message, """\
```

Comment:
Might also be worth separating this out with sections?  
"************************************"?

or ----------?